src/main/java/com/lorepo/icplayer/client/PlayerController.java

```diff
@@ -737,6 +742,17 @@ public native int getIFrameScroll (PlayerController x) /*-{
 		}, false);
 	}-*/;
 
+	public native void handleCurrentPageIdRequest (PlayerController x) /*-{
+		$wnd.addEventListener('message', function (event) {
```

Comment:
Nie jest kasowany ten eventListener z $wnd. Jednak biorąc pod uwagę, że nie widzę po stronie playera w kodzie java kasowanie tego typu eventów to chyba nie stanowi to problemów.